managers/restore-manager/RestoreTaskPoller.js

```diff
@@ -0,0 +1,91 @@
+'use strict';
+
+const _ = require('lodash');
+// const Promise = require('bluebird');
+const eventmesh = require('../../data-access-layer/eventmesh');
+const CONST = require('../../common/constants');
+// const errors = require('../../common/errors');
+const logger = require('../../common/logger');
+// const config = require('../../common/config');
+const catalog = require('../../common/models').catalog;
+const RestoreService = require('./');
+
+
+class RestoreTaskPoller {
+
+  static start() {
+    function poller(object, interval) {
```

Comment:
Do we need to handle HA scenario here. In current implementation I see that both the processes in case of HA will run these pollers

Reply:
Makes sense to handle HA scenario.